frontend/components/Auction.vue

```diff
@@ -305,7 +317,10 @@ export default Vue.extend({
     computed: {
         errorText(): string | null {
             if (!this.isAuctionsLoading && !this.auction) {
-                this.$emit('fetchFinishedAuction', this.auctionId);
+                this.$emit('fetchTakeEventsFromAuction', this.auctionId);
```

Comment:
I am not sure that it's optimal to emit something from the computed property. Are you sure it's emitted once and not every now and then?

Reply:
I moved it to a watcher!